### ISSUE TITLE
Miners upgrading to 1.4.10

### ISSUE THREAD

**Issue by arvicco** [ethereumproject/go-ethereum#3]
Please see https://www.reddit.com/r/EthereumClassic/comments/4ti33y/classic_miners_please_use_geth_149_for_now/d5hld7n

Because we cannot guarantee that all original chain miners move to 1.4.10, there is a possibility of spoofing attack. So, seems like upgrading only after block 192010 (when we can check that no spoofing happened) seems like a better choice, right?

**Comment by arvicco** [comment]
@whatisgravity 

Please see https://www.reddit.com/r/EthereumClassic/comments/4ti33y/classic_miners_please_use_geth_149_for_now/d5hld7n

Because we cannot guarantee that all original chain miners move to 1.4.10, there is a possibility of spoofing attack. So, seems like upgrading only after block 192010 (when we can check that no spoofing happened) seems like a better choice, right?

**Comment by whatisgravity** [comment]
I found the best way to solve this, by adding the fork block "0x4985f5ca3d2afbec36529aa96f74de3cc10a2a4a6c44f2157a57d2c6059a11bb" to bad hashes. It will keep us our on side of the fork when syncing.

**Comment by whatisgravity** [comment]
You can find the commit here:

https://github.com/ethereumproject/go-ethereum/commit/2af3ea164147133afaca90c684891e5d89227ccf

**Comment by whatisgravity** [comment]
The client is ready for production use, you can now build it. 

I'm actively building clients for each operating system. These will be signed and made available soon as they finish. I'll close out this ticket once the builds are released.

**Comment by whatisgravity** [comment]
Builds are available, open a new ticket if you find any issues.
